service/src/main/java/bio/terra/externalcreds/util/ProviderUtils.java

```diff
@@ -1,15 +1,14 @@
 package bio.terra.externalcreds.util;
 
-import bio.terra.externalcreds.generated.model.Provider;
-import java.util.Set;
+import static bio.terra.externalcreds.services.JwtUtils.GA4GH_PASSPORT_V1_CLAIM;
+
+import bio.terra.externalcreds.config.ProviderProperties;
 
 public class ProviderUtils {
-  private static Set<Provider> fenceProviders =
-      Set.of(Provider.FENCE, Provider.DCF_FENCE, Provider.KIDS_FIRST, Provider.ANVIL);
 
   private ProviderUtils() {}
 
-  public static boolean isFenceProvider(Provider provider) {
-    return fenceProviders.contains(provider);
+  public static boolean isPassportProvider(ProviderProperties providerProperties) {
+    return providerProperties.getScopes().contains(GA4GH_PASSPORT_V1_CLAIM);
```

Comment:
As Doug mentioned in https://github.com/DataBiosphere/terra-external-credentials-manager/pull/253#discussion_r1984052010, we might want a different way to identify which providers create passports, such as a new flag in config. This seems ok for now and hopefully this is encapsulated enough to make it easy to change later.